doc/cli.markdown

```diff
@@ -369,20 +369,21 @@ application name
 
 application device type (Check available types with `balena devices supported`)
 
-## app purge &#60;name&#62;
+## app purge &#60;nameOrSlug&#62;
 
 Purge data from all devices belonging to an application.
 This will clear the application's /data directory.
 
 Examples:
 
 	$ balena app purge MyApp
+	$ balena app purge myorg/myapp
 
 ### Arguments
 
-#### NAME
+#### NAMEORSLUG
 
-application name or numeric ID
+application name or org/name slug
```

Comment:
I've raised some matters / questions in flowdock related to documentation of slugs vs IDs:
https://www.flowdock.com/app/rulemotion/i-cli/threads/k9NEA-BfE0IbigEuQL85f0z3mZv